app/server/src/features/accounts/methods.unit.test.ts

```diff
@@ -0,0 +1,463 @@
+import bcrypt from 'bcryptjs';
```

Comment:
Could you arrange the tests in this file using AAA (linked in a previous comment)

Reply:
Do you mean like adding comments of each stage to all the tests? I tried my best to arrange them in that pattern but please point out if there are any you think are not following the pattern correctly.

Reply:
Will do that for a few